src/styles/styles.css

```diff
@@ -151,3 +151,32 @@ h2 {
 .address-card h2 {
   font-size: 1.15rem;
 }
+
+.loader {
```

Comment:
Did you upload an image and make it spin? I did that at first and changed it to a react spinner. It was way easier.

Reply:
tell me how please